staging/src/k8s.io/kubectl/pkg/util/templates/markdown.go

```diff
@@ -58,7 +58,9 @@ func (r *ASCIIRenderer) RenderNode(w io.Writer, node *blackfriday.Node, entering
 		lines := []string{}
 		for _, line := range strings.Split(string(node.Literal), linebreak) {
 			trimmed := strings.Trim(line, " \t")
-			indented := r.Indentation + trimmed
+			// Adding 4 times of indentation will let blackfriday to accept
+			// this literal as Code or CodeBlock again in next invocation
+			indented := r.Indentation + r.Indentation + r.Indentation + r.Indentation + trimmed
```

Comment:
probably easier read with
`strings.Repeat(r.Indentation, 4) + trimmed`

Reply:
Thanks for the suggestion @mpuckett159. Updated.